app/assets/javascripts/govuk_publishing_components/analytics-ga4/ga4-page-views.js

```diff
@@ -148,7 +148,9 @@ window.GOVUK.analyticsGa4.analyticsModules = window.GOVUK.analyticsGa4.analytics
       var content = document.getElementById('content')
       var html = document.querySelector('html')
       if (content) {
-        return content.getAttribute('lang') || this.nullValue
+        if (content.getAttribute('lang')) {
+          return content.getAttribute('lang')
```

Comment:
Since you're calling this twice, you should probably store it in a variable.

Reply:
Whoops, good spot :+1: